src/index.js

```diff
@@ -0,0 +1,185 @@
+import './style.css';
+import count from './count';
+
+function modal(item, iiif, imageId, id) {
```

Comment:
- [ ] You should always use arrow function definition unless you need to use `this` keyword in functions context.
Please refactor your function definitions with arrow functions.